pkg/event_processor/processor_test.go

```diff
@@ -68,12 +71,23 @@ func TestEventProcessor_Serve(t *testing.T) {
 	}
 
 	tick := time.NewTicker(time.Second * 3)
-	select {
-	case <-tick.C:
-	}
+	<-tick.C
+
 	err = ep.Close()
+	lines = strings.Split(buf.String(), "\n")
+	ok := true
+	for _, line := range lines {
+		if strings.Contains(strings.ToLower(line), "dump") {
```

Comment:
Why is a for loop needed to determine whether the string contains `dump`? Which test file contains the string `dump`?

Reply:
In the unit test here, we store all the output from the logger and try to extract string that indicate error. In this case, things like `DumpResponse error: unexpected EOF` is our target. If the any line contains it the this test fails and the exact message is printed out.